staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion.go

```diff
@@ -240,14 +248,19 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 		peerPkgs := extractTag(pkg.Comments)
 		if peerPkgs != nil {
 			klog.V(5).Infof("  tags: %q", peerPkgs)
+			if len(peerPkgs) == 1 && peerPkgs[0] == "false" {
+				peerPkgs = nil
```

Comment:
Because the else is here, this looks wrong.  I would expect this to be an if block before the `if peerPkgs != nil`

Reply:
This wouldn't work - I added a comment here to explain the purpose.

Reply:
ok ,thanks